### PR TITLE
feat: test on any org

### PR DESCRIPTION
**Pull request by ThibaultFy**
Signed-off-by: ThibaultFy <50656860+ThibaultFy@users.noreply.github.com>

## Related issue

`#` followed by the number of the issue

## Please check if the PR fulfills these requirements

- [ ] If the feature has an impact on the user experience, the [changelog](https://github.com/substra/substrafl/blob/main/CHANGELOG.md) has been updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The commit message follows the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification

### CODE REVIEW

**Comment by ThibaultFy** [comment]
/e2e --tests=substrafl --benchmark=camelyon --mode=standalone

**Comment by Owlfred** [comment]
Something happened while processing your message: error: unknown option '--benchmark=camelyon'

**Comment by ThibaultFy** [comment]
/e2e --tests=substrafl --benchmarks=camelyon --mode=standalone

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: substrafl/evaluation_strategy.py [hunk 1]
```diff
@@ -97,6 +97,15 @@ def __init__(
             elif not all(r >= 0 for r in rounds):
                 raise ValueError(f"rounds can be only positive, rounds={rounds} found")
 
+    @property
+    def test_data_nodes_org_ids(self):
+        """Property to get the ids or test data nodes organizations.
+
+        Returns:
+             set: set of organization ids
```
Comment (oleobal) [inline substrafl/evaluation_strategy.py:105]: Out of curiosity, why don't you annotate the return type? (`-> set`)

Comment (ThibaultFy) [reply substrafl/evaluation_strategy.py:105]: It was not on purpose, I'm updating with it ! thx :)

**Review by oleobal — APPROVED** [review]

**Pull request merged**